azureeyemodule/app/iot/iot_interface.cpp

```diff
@@ -12,6 +12,7 @@
 #include <sstream>
 #include <thread>
 #include <unordered_map>
+#include <gst/gst.h>
```

Comment:
Did we actually need this header file for something in here? I don't think I saw anything that would require it, but I could be mistaken.